### ISSUE TITLE
Redirect Manager : Provide Option for Ignore case when applying a redirect rule for "Source" field  or ( [NC] flag equivalent of apache)

### ISSUE THREAD

**Issue by thcharan** [Adobe-Consulting-Services/acs-aem-commons#3150]
THIS IS A FEATURE REQUEST AND NOT AN BUG.

## Required Information

* [ ] AEM Version, including Service Packs, Cumulative Fix Packs, etc: _latest______
* [ ] ACS AEM Commons Version: _latest____
* [ ] Reproducible on Latest? yes

### Expected Behavior

When i want to setup rules that works ignoring case, we do not have an option.

For example -

If i have to setup rules in ACS Redirect Manager tool to handle below use case where -
www.xyz.com/nj
www.xyz.com/NJ

redirects to 
www.xyz.com/summer-events-in-new-jersey

I have to create two entries, or in other words, it can be many combinations based on real world situation.

This feature is equivalent to using [NC] flag In apache, when we setup redirects in apache.

In summary, In the Redirect Manager rule dialog , provide a option for redirects tool user to select "ignore case" , and apply relevant logic in backend when resolving redirects.

### Actual Behavior

No ignore case available for the source field.

**Comment by thcharan** [comment]
@YegorKozlov @davidjgonzalez - fyi

**Comment by YegorKozlov** [comment]
@thcharan this make sense. 
I'll try to make it in next release.
